src/scripts/destinyTrackerApi/reviewDataCache.js

```diff
@@ -0,0 +1,86 @@
+import _ from 'underscore';
+import { itemTransformer } from './itemTransformer.js';
+
+class reviewDataCache {
```

Comment:
I'd pondered that, but in-memory cache lets me skirt around another class of cache invalidation (mostly, tracking when we added data on a finer-grained level).

Once we fetch scores for items, we don't evacuate them (we'll fetch new items, but I don't expect the scores to change much over the course of a few hours).  Review data (user as well as community) gets evacuated after submission, but not otherwise.